packages/beacon-node/test/utils/blsToExecutionChange.ts

```diff
@@ -0,0 +1,16 @@
+import {capella} from "@lodestar/types";
+
+export function generateEmptyBlsToExecutionChange(): capella.BLSToExecutionChange {
```

Comment:
Please do not add this functions, just use `ssz.capella.BlsToExecutionChange.defaultValue()`